modules/microshift-greenboot-dir-structure.adoc

```diff
@@ -6,13 +6,13 @@
 [id="microshift-greenboot-dir-structure_{context}"]
 = How greenboot uses directories to run scripts
 
-Health check scripts run from four `/etc/greenboot` directories. These scripts run in alphabetical order. Keep this in mind when you configure the scripts for your workloads.
+Health check scripts run from two directories. These scripts run in alphabetical order. Keep this in mind when you configure the scripts for your workloads.
```

Comment:
Well, we didn't know it at the time of writing previous doc, but four is actually correct:
- /usr/lib/greenboot/checks/required.d
- /usr/lib/greenboot/checks/wanted.d
- /etc/greenboot/checks/required.d
- /etc/greenboot/checks/wanted.d